owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/settings/SettingsSecurityViewModel.kt

```diff
@@ -52,4 +54,6 @@ class SettingsSecurityViewModel(
 
     fun setPrefTouchesWithOtherVisibleWindows(value: Boolean) =
         preferencesProvider.putBoolean(SettingsSecurityFragment.PREFERENCE_TOUCHES_WITH_OTHER_VISIBLE_WINDOWS, value)
+
+    fun isSecurityEnforcedEnabled(context: Context) = context.resources.getBoolean(R.bool.passcode_enforced)
```

Comment:
OMG, don't pass `Context` as parameter to VM methods! 😁  Inject contextProvider in the ViewModel, just like in `SettingsMoreViewModel`, for example, and get the boolean from there. Otherwise, it will be almost impossible to test. By the way, talking about tests, I don't see unit tests, have you forgotten them? 😈

Reply:
Oops! Good catch! Thank u 😉